pyfar/dsp/dsp.py

```diff
@@ -351,6 +351,70 @@ def regularized_spectrum_inversion(
     return inverse
 
 
+def interpolate_spectrum(frequency_data, n_samples, sampling_rate, method,
+                         kind, fscale='linear', clip=False):
+    """
+    Interpolate incomplete spectrum to complete single sided spectrum.
+
+    TODO: Add references to minimum and linear phase functions
+
+    Parameters
+    ----------
+    frequency_data : FrequencyData
+        Input data to be interpolated
+    n_samples : int
+        The lengths of the corresponding time signal in samples.
+    sampling_rate : int
+        The sampling rate in Hz
+    method : string
+        Specifies the input data for the interpolation
+
+        ``'complex'``
+            Separate interpolation of the real and imaginary part
+        ``'abs_unwrap'``
+            Separate interpolation if the absolute values and unwrapped phase
+        ``'abs_linear'``
+            Interpolation of the absolute values and generation of a linear
+            phase response with a group delay of ``n_samples / 2``
```

Comment:
I think it would be nice to have a parameter for the group_delay, either as separate parameter or as tuple `method=('abs_linear', group_delay)`